openbook/urls.py

```diff
@@ -137,8 +139,14 @@
     path('search/', SearchFollowings.as_view(), name='search-followings'),
 ]
 
+auth_user_subscriptions_patterns = [
+    path('', UserSubscriptions.as_view(), name='user-subscriptions'),
+    path('search/', SearchUserSubscriptions.as_view(), name='search-user-subscriptions'),
+]
+
 auth_patterns = [
     path('', include(auth_auth_patterns)),
+    path('user-subscriptions/', include(auth_user_subscriptions_patterns)),
```

Comment:
Can we remove all the things we're not using in the frontend for now? We can always look back on these commits. Otherwise we'll have code in the codebase no one is sure how or where it's used.